ImageConverter/ImageConverter/Image.cs

```diff
@@ -2,8 +2,10 @@
 
 public class Image
 {
-    private Pixel[,] Pixelmap { get; }
-
+    public Pixel[,] Pixelmap { get; }
+    public int ColorMaxValue { get => 255; }
```

Comment:
IDK, this field describes how deep it will be colour so maybe it can be replaced with some kind of bit depth for each color pixel. But I really don`t know why we don`t want to store info like this, that describes image more.